mmv1/third_party/terraform/website/docs/r/compute_instance_template.html.markdown

```diff
@@ -480,4 +494,4 @@ $ terraform import google_compute_instance_template.default {{name}}
 ```
 
 [custom-vm-types]: https://cloud.google.com/dataproc/docs/concepts/compute/custom-machine-types
-[network-tier]: https://cloud.google.com/network-tiers/docs/overview
\ No newline at end of file
+[network-tier]: https://cloud.google.com/network-tiers/docs/overview
```

Comment:
Unsure what has changed here.

Reply:
Probably a newline/whitespace.

Reply:
Highlighting is my friend! This was whitespace at EOL 👍